src/utilities/dom.ts

```diff
@@ -75,7 +75,8 @@ export function getVirtualParent(child: HTMLElement): HTMLElement {
  * @returns {HTMLElement}
  */
 export function getParent(child: HTMLElement, allowVirtualParents: boolean = true): HTMLElement {
-  return child && (allowVirtualParents && getVirtualParent(child) || child.parentElement);
+  return child && (allowVirtualParents && getVirtualParent(child) || child.parentElement ||
+    (child.parentNode && child.parentNode.nodeType === Node.ELEMENT_NODE ? child.parentNode as HTMLElement : null));
```

Comment:
Hmm. What if we simplify this:

```ts
return child && (
   allowVirtualParent && getVirtualParent(child) ||
   child.parentNode && child.parentNode as HTMLElement
);
```

I can't really come up with valid scenarios here where the extra logic is necessary.

Also, it would be great to add a test here in dom.test.js. It's easy to set up.